src/content/protocols/ethena/usde.md

```diff
@@ -0,0 +1,329 @@
+        status: "fixed",
+      },
+    ],
+    [
+      {
+        text: "Upgrades with potential of “loss of funds or unclaimed yield” not protected with onchain governance AND Exit Window >= 30 days",
+        status: "unfixed",
+      },
+      {
+        text: "Dependencies with High or Medium centralization score and no mitigations.",
+        status: "unfixed",
+      },
+      { text: "Alternative third-party frontends exist", status: "fixed" },
+    ],
+  ]
+---
+
+# Summary
+
+Ethena is a synthetic dollar protocol built on Ethereum. Ethena's synthetic dollar, `USDe`, is fully backed by crypto assets (Bitcoin, Ethereum, Solana), yield from staked ETH through LSTs and earnings from funding rates from short futures positions on CEXs. `USDe` is minted and redeemed by whitelisted KYCed users. Any user can acquire `USDe` on the open market and stake it to get access on yield sourced from the strategies and passed onto the users of the `USDe` stablecoin. `ENA` the governance token is freely tradeable and can be used to signal the direction of the protocol through off-chain governance on Snapshot.
+
+# Ratings
+
+## Chain
+
+The report is concerned with the USDe stablecoin protocol deployed on Ethereum mainnet. Ethereum mainnet achieves a _Low_ centralization risk score.
+
+> Chain score: Low
+
+## Upgradeability
+
+The right to mint new `USDe` can be transferred to any address, and there are no safeguards to limit how much can be issued. If the minter role is transferred to a malicious or compromised address, they could create unbacked `USDe`, `USDe` holders would suffer losses through dilution and unbacked emission of `USDe` (_loss of funds_).
+
+Collateral funds transferred from the user into the Ethena protocol are sent to addresses labelled as custodian addresses. These addresses are EOAs. If the addresses are not set correctly or compromised, collateral deposited by users could be lost immediately and would lead to _loss of funds_.
+
+Users can be removed from the whitelist for minting and redeeming. If users ,that already minted `USDe`, are removed from the whitelist, they have to sell their `USDe` on the open market since they have no access to their deposited collateral which could result in the _loss of funds_ if the `USDe` price drops below the collateral value.
+
+The `USDe` staking contract which allows users to stake `USDe` includes a blacklist feature. If a _Staker_ is blacklisted, they are prevented from staking or unstaking `USDe`. This could result in _loss of funds_ if users are unable to retrieve their staked `USDe` and and if they have to sell `sUSDe` on the open market at a discount. Blacklisting can lead to a complete _loss of funds_ for a blacklisted user if their `sUSDe` is burned from their wallet and redistributed to another address or equally to all _Stakers_. Blacklisting and seizing staker funds are two steps that are carried out by two permission owners.
+
+> Upgradeability score: High
```

Comment:
Here I think you should also mention how strategies are assigned and how can they be changed?